mesonbuild/wrap/wrap.py

```diff
@@ -89,6 +90,12 @@ class WrapException(MesonException):
 class WrapNotFoundException(WrapException):
     pass
 
+class WrapPatchFormatException(WrapException):
+    pass
+
+class PatchUnpackException(WrapException):
+    pass
+
```

Comment:
I don't think it's worth adding specialised exceptions if you don't catch them anywhere. WrapNotFoundException above is because we need to catch that specific case for special treatment in interpreter IIRC.